plugins/python-build/share/python-build/3.10.2

```diff
@@ -0,0 +1,10 @@
+#require_gcc
+prefer_openssl11
```

Comment:
Do we still need to prefer OpenSSL 1.1?
[AFAICS](https://docs.python.org/3/whatsnew/changelog.html), 3.10 supports 3.0.